### DISCUSSION TITLE
Feature Request: Field headers

### DISCUSSION THREAD

**Discussion by ghost** [craftcms/cms#8715] [Ideas]
### Description
It would be nice to have field headers that work similar to the source headers on index pages. You are able to manage field headers via the field layout designer. The field headers are visible on the element edit pages. You can use them to group fields that belong together. This would be another way to organise your fields inside your tabs and is especially useful for (commerce) variants because they can't have tabs. You could use this to for example group address fields inside a tab for contact information.

**Comment by brandonkelly** [comment]
We’re planning something like this for Craft 4, along with a lot of other field layout designer improvements :)

**Comment by spencech-upbeat** [comment]
I think this is what I was just about to file a feature request for. I've got a matrix block with a set of field instructions that repeats with every block addition. This gets pretty redundant and clunky after a few entries. It would be great to be able to insert an Instructions style block, effectively a header I guess, that explains the subsequent fields. Attached is what I have currently. Only concern re: headers specifically would be legibility for longer text (i.e., bold and blocky is harder to read for paragraphs than short text strings I think).
<img width="1428" alt="Screen Shot 2020-06-26 at 8 41 52 AM" src="https://user-images.githubusercontent.com/65023226/85858243-4c7ec380-b789-11ea-919d-6686555d7197.png">
